odoo/addons/base/models/ir_ui_view.py

```diff
@@ -586,7 +586,12 @@ def apply_inheritance_specs(self, source, specs_tree, inherit_id):
                         source = copy.deepcopy(spec[0])
                     else:
                         for child in spec:
-                            node.addprevious(child)
+                            new_node = child
+                            if child.get('position') == 'move':
```

Comment:
We could easily implement a "duplicate" or "copy" function. A version of move without removing the element.
I can see a lot of use for something like that.

Reply:
you cannot have 2 times the same field.
The framework doesn't support it.

Reply:
And why would it, as it encourages awkward UI design and entails lots of complications in terms of implementation and synchronization?

Reply:
yes i was not thinking about fields. but other types of nodes. i agree with the field observation and the fact it would create harm. even if a copy could be interesting with other types of elements , it's misuse with fields is obvious. i retire my proposal.

Reply:
@JKE-be 
Apparently it does since version 11 and the refactoring the Javascript framework by @ged-odoo and his team, at least this was my impression and it is working great for some fields I use with my clients (for example, sometimes we need to expose fields which are hidden b/c of a group a user is not part of). Of course it should be used with explicit care and thoughts as @odony has a more than valid point.

Reply:
@wtaferner Duplicate fields in views actually more or less work, but it is more a happy side effect than a design goal.  Some things will probably be unexpected.  For example, mixing modifiers/attributes, or disabling onchange in one of them but not in the other.  Or having a domain or a specific context.  Or being use as a target for an xpath. Anyway, I guess that they should work fine for most simple usecases, but we definitely do not officially support multiple fields in the same view.

Reply:
@ged-odoo 
Thx for clarification. I somehow already assumed that it is not officially supported reading the comments of Jeremy and Olivier. Anyway, it was super helpful for my use cases ;-) I will therefore find other solutions for these kind of issues in the future.

Reply:
@wtaferner  why not add a compute that depends on field A called A_bis and use that as a "duplicate". the compute would assign in active record pattern interface self.A_bis = self.A. so you would have no wierdness with onchanges and possible XPATH ill effects.